js/web/lib/onnxjs/ops/depth-to-space.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {Attribute} from '../attribute';
```

Comment:
do you need to update file backends\webgl\op-resolve-rules.ts?

Reply:
good catch, missed that file while porting.